translations/strings.yaml

```diff
@@ -310,6 +310,8 @@ settings:
     value: Connect VPN on startup
     comment: Startup refers to a device's startup, not the VPN client's.
   startAtBootSubtitle: Mozilla VPN will launch and connect when you start up your device
+  tunnelPort53: Tunnel VPN using port 53/DNS
```

Comment:
Question for copy: we already have the details in the description, do we need to repeat the port number here? `Tunnel VPN through DNS port` seems clear enough.

From a localization perspective, if we end up using this form, I would only ask to add a comment that "tunnel" is a verb, as in "direct traffic through".

Reply:
Good suggestions, thanks